tests/Functional/PhpunitTest.php

```diff
@@ -26,6 +26,24 @@ public function testPreviousChangesAreRolledBack(): void
         $this->assertTrue($this->connection->isTransactionActive());
     }
 
+    /**
+     * @dataProvider someDataProvider
+     */
+    public function testWithDataProvider(string $argument): void
+    {
+        $this->assertRowCount(0);
+        $this->insertRow();
+        $this->assertTrue($this->connection->isTransactionActive());
```

Comment:
can we maybe use the string `$argument` for the message here? like 
```suggestion
        $this->assertTrue($this->connection->isTransactionActive(), $argument);
```

so at least its not a completely unused variable

Reply:
Done!